edxadapt/static/js/src/edxadapt.js

```diff
@@ -65,8 +65,8 @@ function EdxAdaptXBlock(runtime, element) {
         })
         .fail(function(jqXHR, textStatus) {
             if (jqXHR.status == 404) {
-                // Student not found in EdxAdapt. Let's register them
```

Comment:
By the way, `them` here is being used as a gender-neutral singular pronoun. Because student can be male or female.